### PR TITLE
refactor: set prefix for env vars

### PR DESCRIPTION
**Pull request by leolara**
Closes https://github.com/privacy-scaling-explorations/zkevm-chain/issues/26

Notes:
 + Now we have PROVERD_PARAMS_PATH and COORDINATOR_PARAMS_PATH I set them to the same value in the "dev" container of docker-compose.
 + There is a VERBOSE in the coordinator tests, I changed it to TEST_VERBOSE because I had the impression this was a cross cutting concern more than a subpackage thing. But let me know if you would like to change it.

### CODE REVIEW

**Comment by leolara** [comment]
@pinkiebell I messaged you in matrix

File: docker-compose.yml [hunk 1]
```diff
@@ -120,16 +120,16 @@ services:
...
-      - L1_BRIDGE=0x936a70c0b28532aa22240dce21f89a8399d6ac60
-      - L1_PRIV=$MINER_PRIV_KEY
-      - PROVER_RPCD_URL=http://prover-rpcd:8545
-      - DUMMY_PROVER=${DUMMY_PROVER:-1}
-      - ENABLE_FAUCET=1
-      - PARAMS_PATH=${PARAMS_PATH:-/testnet/}
+      - COORDINATOR_LISTEN=[::]:8545
+      - COORDINATOR_RPC_SERVER_NODES=server-testnet-geth:8545
+      - COORDINATOR_L2_RPC_URL=http://leader-testnet-geth:8545
+      - COORDINATOR_COORDINATOR_L1_RPC_URL=http://l1-testnet-geth:8545
```
Comment (pinkiebell) [inline docker-compose.yml:126]: Double prefix

File: docker-compose.yml [hunk 2]
```diff
@@ -159,20 +159,21 @@ services:
...
-      - L1_PRIV=$MINER_PRIV_KEY
-      - DUMMY_PROVER=${DUMMY_PROVER:-1}
-      - ENABLE_FAUCET=1
+      - COORDINATOR_LISTEN=[::]:8000
+      - COORDINATOR_RPC_SERVER_NODES=server-testnet-geth:8545
+      - COORDINATOR_L2_RPC_URL=http://leader-testnet-geth:8545
+      - COORDINATOR_L1_RPC_URL=http://l1-testnet-geth:8545
+      - COORDINATOR_L1_BRIDGE=0x936a70c0b28532aa22240dce21f89a8399d6ac60
+      - COORDINATOR_L1_PRIV=$MINER_PRIV_KEY
+      - COORDINATOR_PROVER_RPCD_URL=http://dev:8001
```
Comment (pinkiebell) [inline docker-compose.yml:168]: Can we move this back to the `useful env vars if running the proverd inside the dev image` .. 🙂

**Review by pinkiebell — CHANGES_REQUESTED** [review]
Just 2 things and we are good to go 🎉

File: docker-compose.yml (hunk 2 above)
Comment (leolara) [reply docker-compose.yml:168]: sorry

**Review by pinkiebell — APPROVED** [review]
LGTM 🦾

**Pull request merged**